### PR TITLE
[18.0][MIG] server_environment_ir_config_parameter: Migration to 18.0

### PR DESCRIPTION
**Pull request by chaule97**

### CODE REVIEW

**Comment by sbidoul** [comment]
See also #202

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@qgroulard fine for you?

File: server_environment_ir_config_parameter/readme/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The migration of this module from 17.0 to 18.0 was financially supported by Camptocamp.
```
Comment (sbidoul) [inline server_environment_ir_config_parameter/readme/CREDITS.md:1]: And Acsone, since I did the review :stuck_out_tongue_winking_eye:

**Review by sbidoul — APPROVED** [review]
LGTM. Code reivew. I compared with 17, it's identical.

**Pull request merged**